src/main/java/com/yugabyte/sample/apps/SqlForeignKeysAndJoins.java

```diff
@@ -93,7 +95,16 @@ public void dropTable() throws Exception {
 
   @Override
   public void createTablesIfNeeded(TableOp tableOp) throws Exception {
-    Connection connection = getPostgresConnection();
```

Comment:
@tedyu can we move the retry logic to `AppBase.getPostgresConnection()`, instead of having to duplicate the code in multiple places. Is there any specific reason for having to make retry logic individually?

Reply:
Also, In `SqlForeignKeysAndJoins.java` there are multiple places where getPostgresConnection() is used where retry won't be called like `getPreparedInsertUser()`, `getPreparedInsertUserOrder()`, `dropTable()` and `AppBase.getPostgresConnection()`. We should move the retry logic to `AppBase.getPostgresConnection()`.